src/renderer/components/+add-cluster/add-cluster.tsx

```diff
@@ -115,10 +116,11 @@ export class AddCluster extends React.Component {
           Read more about adding clusters <a href={`${docsUrl}/clusters/adding-clusters/`} rel="noreferrer" target="_blank">here</a>.
         </p>
         <div className="flex column">
-          <AceEditor
-            autoFocus
-            showGutter={false}
-            mode="yaml"
+          <MonacoEditor
+            options = {{ automaticLayout: true, minimap: {enabled: false}, lineNumbers: "off"}}
```

Comment:
Oooh, that would be some nice user preferences (for a future PR).

Reply:
Agree, I'm working on it right now